Formula/dotnet.rb

```diff
@@ -51,6 +51,23 @@ def install
     (bin/"dotnet").write_env_script libexec/"dotnet", DOTNET_ROOT: libexec
   end
 
+  def caveats
+    <<~EOS
+      To ensure the Homebrew installed version of .NET Core is used, you have to
+      add the following line to your ~/.bash_profile, ~/.zshrc or
+      ~/.config/fish/config.fish file:
+
+        export DOTNET_ROOT="#{opt_libexec}"
```

Comment:
This is already done by line 51 (which wraps the dotnet binary with a script that does that)

Reply:
That doesn't help other software needing to find where .NET is installed.